CHANGELOG.md

```diff
@@ -1,6 +1,40 @@
 ## [Unreleased]
 
-### ...
+### Features
+
+...
+
+## 1.3.0 (1.10.2019)
+
+### Features
+
+#### ([1110](https://github.com/allegro/hermes/pull/1110)) Flat binary storage for consumers workload
```

Comment:
Is it ok that this and next statements have same PR link? maybe we can group them.

Reply:
Fixed